components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -1507,4 +1770,200 @@ private void setSPAttributeToRequest(HttpServletRequest req, String issuer, Stri
                     tenantDomain, e);
         }
     }
+
+    private void doFrontChannelSLO(HttpServletResponse response,
+                                   SAMLSSOServiceProviderDO samlssoServiceProviderDO, String sessionIndex,
+                                   String originalLogoutRequestIssuer, String originalIssuerLogoutRequestId,
+                                   boolean isIdPInitSLO, String relayState, String returnToURL)
+            throws IdentityException, IOException {
+
+        SessionInfoData sessionInfoData = SAMLSSOUtil.getSessionInfoData(sessionIndex);
+        String subject = sessionInfoData.getSubject(samlssoServiceProviderDO.getIssuer());
+
+        LogoutRequest logoutRequest = SAMLSSOUtil.buildLogoutRequest(samlssoServiceProviderDO, subject, sessionIndex);
+        storeFrontChannelSLOParticipantInfo(samlssoServiceProviderDO, originalLogoutRequestIssuer, logoutRequest,
+                originalIssuerLogoutRequestId, sessionIndex, isIdPInitSLO, relayState, returnToURL);
+
+        // TODO: UI configuration to check for the binding and filter.
```

Comment:
If this is already finished shall we remove this?

Reply:
This is completed and the todo is removed with #208